docs/WifiPerformance.md

```
@@ -60,980 +60,912 @@
-    ```
+```
+    freq: 2462.0
+    signal: -39 dBm
+    tx bitrate: 174.0 MBit/s
+```
 
-### Intel AX200
+#### EDUP EP-AC1681
 
-<img src=https://stuff.armbian.com/netbox/media/devicetype-images/AX200.png>
+<img src=https://stuff.armbian.com/netbox/media/devicetype-images/RTL88x2BU.png>
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)